build/ios/mojom/public/base/base_values.mm

```diff
@@ -334,12 +334,13 @@ - (nullable id)jsonObject {
 
 base::Value BaseValueFromNSDictionary(
     NSDictionary<NSString*, MojoBaseValue*>* dictionary) {
-  base::Value dict(base::Value::Type::DICTIONARY);
+  base::Value result(base::Value::Type::DICT);
+  base::Value::Dict& dict = result.GetDict();
```

Comment:
@cdesouza-chromium Isn't this still a deprecated usage? Do we need to update these properly?

Reply:
No, we are still allowed to create `Value` types with a specific type, including DICT. We have replaced them in most places, because that was appropriate, but for this particular case, we do want to move about a `base::Value`, rather than a `base::Value::Dict`.

The key `base::Value::Type::DICTIONARY` got deleted upstream though.